chore: seed developer 16.0 help (#614)
keymanapp/help.keyman.com: developer/16.0/context/server.md

---
title: Keyman Developer Server
---

# Introduction

Keyman Developer Server provides a web-based front end for testing and sharing
keyboards, lexical models and packages.

The primary benefits of Keyman Developer Server are:

* Test keyboards in a browser on your computer, including simulation of mobile
  devices and touch keyboards
* Test lexical models in a browser on your computer
* Test keyboards, lexical models and packages on mobile devices
* Share keyboards, lexical models and packages to mobile devices and other users

> Keyman Developer Server is new in Keyman Developer 15, and replaces the old Web

Reviewer: I guess this was the only reference to Keyman Developer 15. Maybe
```suggestion
Keyman Developer Server was added in Keyman Developer 15, and replaces the old web
```
